### PR TITLE
Homework # 5

### PR DESCRIPTION
**Pull request by SherstianoyVolchara**
1. added first and second classes 
2. added int variable and  method to second class
3. instantiation instance of the 2nd class in the 1st class

### CODE REVIEW

File: src/main/java/First.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class First {
+    public static void main(String[] args) {
+        Second second = new Second();
+        System.out.println("~~~" + second.showNumber() + "~~~");
```
Comment (taraskarpenko) [inline src/main/java/First.java:4]: Примените пожалйста форматирование строк

File: src/main/java/Second.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+public class Second {
+    public int showNumber() {
+        int number = 574;
+        System.out.println("---" + number + "---");
```
Comment (taraskarpenko) [inline src/main/java/Second.java:4]: Примените пожалйста форматирование строк

**Review by taraskarpenko — CHANGES_REQUESTED** [review]

File: src/main/java/First.java (hunk 1 above)
Comment (SherstianoyVolchara) [reply src/main/java/First.java]: @taraskarpenko added

File: src/main/java/Second.java (hunk 2 above)
Comment (SherstianoyVolchara) [reply src/main/java/Second.java]: @taraskarpenko added

**Review by taraskarpenko — APPROVED** [review]

**Pull request merged**